src/main/main.cpp

```diff
@@ -48,8 +54,13 @@ int main(int argc, char *argv[])
 
     QGuiApplication app(argc, argv);
 
+    #ifdef Q_OS_MACOS
+    MacOsEventListener evListener;
+    initMacOsEventListener(&evListener);
```

Comment:
@tim-sueberkrueb by default QtCreator offered don't indent preprocessor directives at all. We can stick to it.